src/Impostor.Api/Innersloth/GameOptions/LegacyGameOptionsData.cs

```diff
@@ -0,0 +1,263 @@
+using Impostor.Api.Innersloth.GameOptions.RoleOptions;
+
+namespace Impostor.Api.Innersloth.GameOptions;
+
+public class LegacyGameOptionsData : IGameOptions
```

Comment:
why do we still want this class?

Reply:
Impostor.Server itself doesn't really need it anymore, but is there any reason to remove it? You could use it parse old gameoptions data on disk or whatever